src/templates/components/HostedHeader.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+	import '$templates/components/fonts/SansBold.css';
+
+	export let logo: string | null;
+</script>
+
+<header>
+	<h1>From our advertisers</h1>
+	{#if logo}
+		<div class="logo">
```

Comment:
'Advertiser content' needs to be flush with the top so it's touching the green border - I believe the padding at the top is coming from the aside
<img width="95" alt="Screenshot 2024-01-11 at 10 49 09" src="https://github.com/guardian/commercial-templates/assets/108270776/3102e1f1-62ae-4c6f-b802-1755cbb0bd42">